test/e2e/tuning_options_test.go

```diff
@@ -320,3 +326,201 @@ func TestHAProxyTimeoutsRejection(t *testing.T) {
 		}
 	}
 }
+
+// TestCookieLen verifies that when logging is enabled and a cookie capture is specified, any captured cookies are
+// truncated at the expected max length.
+func TestCookieLen(t *testing.T) {
+	testCases := []struct {
+		name      string
+		maxLength int
+	}{
+		// 63 bytes was the limit before OCPBUGS-3356 was fixed; This should pass even without the patch, so it's more
+		// of a sanity check.
+		{"63 bytes", 63},
+		// The lower bound. 64 bytes is the smallest number that would be truncated short prior to the OCPBUGS-3356 fix.
```

Comment:
How can both 63 and 64 be the lower limit?

Reply:
63 bytes was the limit before the fix in openshift/router#436. You could remove that case and it would cover the range where behavior has changed due to the router PR, but I think it's helpful to include the 63 byte case as a sanity check. 
If it passes but all the other cases fail, there's something wrong with the way the global variable `tune.http.cookielen` was set, but if it and the other cases fail, it indicates that the failure is probably unrelated to `tune.http.cookielen`.

Reply:
I'm going to suggest some re-wording of the comments now that I've reviewed https://github.com/openshift/router/pull/436

Reply:
Thanks for your suggestions. I did adjust the wording slightly, but I removed any reference to the specific jira issue. Let me know if you think they are in need of any additional edits.